### ISSUE TITLE
create-react-app-typescript is deprecated

### ISSUE THREAD

**Issue by ruslanegorov** [Microsoft/TypeScript-React-Starter#246]
https://github.com/wmonk/create-react-app-typescript

README says that users need to migrate to create-react-app

**Comment by thu-san** [comment]
Agreed. Should update to `npx create-react-app my-app --typescript`
https://facebook.github.io/create-react-app/docs/adding-typescript

`--scripts-version=react-scripts-ts` doesn't work quite well with Scss.

**Comment by JukoPowel** [comment]
I created similar issue with article how to use CRA for TS and migrate from wmonk solution, so this one may be closed.

**Comment by JulianKowalczuk** [comment]
I close my issue. Link to useful guide here:
https://vincenttunru.com/migrate-create-react-app-typescript-to-create-react-app/

**Comment by datenreisender** [comment]
Outdated docs are a pain and even worse when it is not obvious that they are outdated. 😕 

I was just confused, that the README still uses the deprecated term SFC, which is misleading since the introduction of hooks. But since this does not seem to be maintained, I will not open an issue for that.

**Comment by PhilParisot** [comment]
Agreed, these docs need to be updated ASAP, very misleading to newbies (like me)

**Comment by PhilParisot** [comment]
Another resource from the React Website: https://reactjs.org/docs/static-type-checking.html#typescript

**Comment by yzhong52** [comment]
It is interesting, there is no active maintainer of this repo. There are a lot of open PRs. This issue should obviously be fixed as well.
